src/js/components/Menu/Menu.js

```diff
@@ -184,20 +184,30 @@ const Menu = forwardRef((props, ref) => {
     }
   };
 
-  const content = children || (
-    <Box
-      direction="row"
-      justify={justifyContent}
-      align="center"
-      pad={theme.button.default ? undefined : 'small'}
-      gap={label && icon !== false ? 'small' : undefined}
-    >
-      <Text size={size}>{label}</Text>
-      {icon !== false
-        ? (icon !== true && icon) || <MenuIcon color={iconColor} size={size} />
-        : null}
-    </Box>
-  );
+  const menuLabel = <Text size={size}>{label}</Text>;
+  const menuIcon =
+    icon !== false
+      ? (icon !== true && icon) || <MenuIcon color={iconColor} size={size} />
+      : null;
+
+  let content;
+  if (children) content = children;
+  else if (!theme.button.default) {
+    content = (
+      <Box
+        direction="row"
+        justify={justifyContent}
```

Comment:
Per our convo, investigating the `justifyContent` + default button scenarion.

Reply:
An update here, currently button does not support ability to control justification of content within the width of the button. However, this could be considered a future enhancement. That being said, the benefits of `justifyContent` on Menu could still be achieved by providing Menu with children, such as:
```
<Menu>
        {() => (
          <Box align="center" gap="small" direction="row">
            <Text>Menu</Text>
            <FormDown />
          </Box>
        )}
</Menu>
```
This is not as seamless as a prop name, but the functionality is still possible. I feel like this is sufficient for now and a discussion of a button enhancement could be a future discussion. What do you think? @halocline